packages/teams-js/src/public/media.ts

```diff
@@ -25,18 +25,13 @@ import {
   validateSelectMediaInputs,
   validateViewImagesInput,
 } from '../internal/mediaUtil';
-import {
-  callCallbackWithErrorOrResultFromPromiseAndReturnPromise,
-  callCallbackWithSdkErrorFromPromiseAndReturnPromise,
-  generateGUID,
-  InputFunction,
-} from '../internal/utils';
+import { generateGUID } from '../internal/utils';
+import { runtime } from '../public/runtime';
 import { FrameContexts, HostClientType } from './constants';
 import { ErrorCode, SdkError } from './interfaces';
-import { runtime } from './runtime';
 
 /**
- * @alpha
+ * @deprecated
```

Comment:
I generated the docs locally and I see that the deprecated tag only appears at the top of the page, and doesn't get applied to all the functions within the namespace. I think we want to add @deprecated to all the exported functions in `media`. 

@erikadoyle is there a recommended way to annotate the deprecated tag (or maybe a different approach to SDK reference documentation) to help developers understand that this file is deprecated as of v2, but we are still working on what to replace it with?

Reply:
Yes, each function needs its own @deprecated annotation in order to render the note. [appInitialization.ts](https://github.com/OfficeDev/microsoft-teams-library-js/blob/2.0-preview/packages/teams-js/src/public/appInitialization.ts) is a good example for this.

Don't think there's any MS Docs guidance for this specific scenario, but how about adding a note like this:

  /**
   * @deprecated
   * As of 2.0.0, use only for backwards compatibility of existing code.
   * Description goes here.
   */

Reply:
Thanks @erikadoyle! That's super helpful. @lakhveerk could you please add the deprecated info as Erika described for each API in the namespace and the namespace itself? For right now, please use version 2.0.0-beta.6 in the comments. I'll be doing another pass before we release 2.0.0.

Reply:
yeah thank you!